src/main/java/reactor/kafka/receiver/internals/ConsumerHandler.java

```diff
@@ -63,7 +62,8 @@
 
     private final ConsumerEventLoop<K, V> consumerEventLoop;
 
-    private final FluxIdentityProcessor<ConsumerRecords<K, V>> processor = Processors.more().multicast(1);
+    @SuppressWarnings("deprecation")
+    private final EmitterProcessor<ConsumerRecords<K, V>> processor = EmitterProcessor.create(1);
```

Comment:
I need `EmitterProcessor` here because later we pass `FluxSink` to `ConsumerEventLoop` that uses `FluxSink#onRequest` / `FluxSink#requestedFromDownstream()`